.circleci/config.yml

```diff
@@ -84,6 +84,30 @@ jobs:
           command: bundle exec erblint --lint-all
       - store_test_results:
           path: ~/rspec
+  generators:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /app
+      - checkout
+      - restore_cache:
+          keys:
+            - bundler-dependencies-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Install ruby dependencies
+          command: bundle install
+      - save_cache:
+          key: bundler-dependencies-{{ checksum "Gemfile.lock" }}
+          paths:
+            - /usr/local/bundle/
+      - run:
+          name: Wait for db
+          command: dockerize -wait tcp://localhost:5432 -timeout 2m
+      - run:
+          name: Run generators RSpec
+          command: mkdir ~/rspec && bundle exec rspec --format progress --format RspecJunitFormatter -o ~/rspec/rspec.xml
```

Comment:
You're running the tests of the main gem, is this intended?

Also, I think you're missing the coverage report generation, did you copy this from another job? I might have forgotten to add this when I moved the coverage to 🤔

Reply:
Ooops! No, absolutely not intended! Let's see what the real timing is now... :grimacing:

Reply:
And yes, I copied it from main and it seems like coverage is not there.

Reply:
Damn. Let's leave it as it is right now, and I'll try to fix it in another PR once this one is merged 😄

Reply:
Ok, I will change the job the run the tests it's supposed to run though :) Hopefully the pass...

Reply:
🤞